draft-scurtescu-secevent-event-stream-management-api.md

```diff
@@ -449,9 +449,9 @@ the Event Receiver as a result of the above request:
   "iat": "1493856000",
   "events": [
     "urn:ietf:params:secevent:event-type:core:verification" : {
-      "state": "VGhpcyBpcyBhbiBleGFtcGxlIHN0YXRlIHZhbHVlLgo=",
```

Comment:
I've seen JavaScript engines choke on trailing commas like that (IE 8? I think? was one example), which is why I removed them. Personally, I'm not bothered by the previous line being changed when we add something, but I can put them back if you'd prefer it.